### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by j-f1**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

*Please explain why you think this Topic page should be curated*:

ESLint is a very popular linter (there’s a big list of companies who use it [on the homepage](https://eslint.org/)).

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by j-f1** [comment]
The logo looks terrible. I’ll upload a new one in a few minutes, then reopen this PR.

File: topics/eslint/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+display_name: ESLint
+github_url: https://github.com/eslint/eslint
+logo: eslint.png
+related: eslint-plugin, eslint-config
+released: June 2013
+short_description: The pluggable linting utility for JavaScript and JSX
+topic: eslint
+url: https://eslint.org/
+---
+ESLint is an extensible static-analysis tool for JavaScript and related languages that helps you catch errors before they bite you in production.
```
Comment (nayafia) [inline topics/eslint/index.md]: I like the `before they bite you` part :smile: but it feels a bit casual for a referential prose style. We also try to avoid "you" references.

Could you rephrase? Actually, even `helps you catch errors before they bite` would be 👍 by me.

Comment (j-f1) [reply topics/eslint/index.md]: How does the new one sound?

File: topics/eslint/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: Nicholas C. Zakas
+display_name: ESLint
+github_url: https://github.com/eslint/eslint
+logo: eslint.png
+related: eslint-plugin, eslint-config
```
Comment (cheshire137) [inline topics/eslint/index.md]: I think because neither of these topics have their own curated content yet, and they're very much dealing with ESLint, you could move them to `aliases` instead. That way, the curated content from this file will show up on the /topics/eslint-plugin and /topics/eslint-config pages.

You could also add linting, linter, and javascript to `related`.

Comment (j-f1) [reply topics/eslint/index.md]: Done! ✅

**Comment by cheshire137** [comment]
Looks like j-f1 made some edits since your last review, @nayafia. 👀

**Comment by nayafia** [comment]
Thank you @j-f1! ✨
